src/pages/TestAccount/TestAccount.js

```diff
@@ -18,6 +21,19 @@ const TestAccount = () => {
     return signatureVerify(signedMessage, signature, hexPublicKey).isValid;
   };
 
+  const login = async () => {
+    const signinMessage = getSigninMessage(username);
+    const signature = await sign(signinMessage);
+    const result = await Parse.User.logIn(username, password, { installationId: JSON.stringify({ signature, address: account }) });
```

Comment:
I don’t see a JSON.parse in parse backend code. We don’t need it when read the params on the backend?